Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/WalletConnect.cs

```diff
@@ -6,42 +6,34 @@
 using AnkrSDK.WalletConnectSharp.Core;
 using AnkrSDK.WalletConnectSharp.Core.Models;
 using AnkrSDK.WalletConnectSharp.Core.Network;
+using AnkrSDK.WalletConnectSharp.Unity.Infrastructure;
 using AnkrSDK.WalletConnectSharp.Unity.Models.DeepLink;
 using AnkrSDK.WalletConnectSharp.Unity.Models.DeepLink.Helpers;
 using AnkrSDK.WalletConnectSharp.Unity.Network;
 using AnkrSDK.WalletConnectSharp.Unity.Utils;
-using Cysharp.Threading.Tasks;
 using UnityEngine;
+using Cysharp.Threading.Tasks;
 using Logger = AnkrSDK.InternalUtils.Logger;
 
 namespace AnkrSDK.WalletConnectSharp.Unity
 {
-	[RequireComponent(typeof(NativeWebSocketTransport))]
-	public class WalletConnect : MonoBehaviour
+	public class WalletConnect : IQuittableComponent, IPausableComponent, IDisposable
```

Comment:
If it is IDisposable - do we handle Disposing ourselves?

Reply:
We are using "using" directive in WalletConnectTests